frontend/src/API/MembersAPI.ts

```diff
@@ -64,4 +64,24 @@ export class MembersAPI {
       member
     ).then((res) => res.data);
   }
+
+  public static checkMemberMeetsCategory(
+    otherMember: IdolMember,
+    category: string
+  ): Promise<boolean | undefined> {
+    const res = APIWrapper.get(
+      `${backendURL}/member/${otherMember.email}/${Buffer.from(category).toString('base64')}`
```

Comment:
hm, this doesn't feel the most idiomatic and will make debugging tricky. Is putting the category name directly here too long? If so, we could maybe consider using some other key that can be easily traced back to the coffee chat category? Something that's more human-readable would make it easier to debug.

Reply:
True. I encoded it because if a category included '/', it messed with getting the request parameters.

Reply:
Try:

```
const category = "category/with/slash";
const encodedCategory = encodeURIComponent(category);
```